bin/wp-merge-to-cp.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# First parameter is the changeset number
+# Second parameter is the WP branch
```

Comment:
Instead of indicating the parameters in comments, the script should make sure they are both present.  If not, it should exit with a help message and error.

Also, it shouldn't be necessary to specify the WP branch where a changeset was merged.  The script can check all the branches we're interested in to find the commit.